core/src/main/java/rdx/works/core/domain/ProfileState.kt

```diff
@@ -32,5 +34,9 @@ sealed class ProfileState {
         fun hasNetworks(): Boolean {
             return profile.hasNetworks
         }
+
+        fun isCurrentNetworkMainnet(): Boolean {
+            return profile.currentGateway.network.id == NetworkId.MAINNET
+        }
```

Comment:
why this? 👀

Reply:
my bad, this was leftover from very early iteration of the feature